meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
Yeah this line was just added because it is another optarg mutation. I'm not sure how different this mutation could be  since the point of this PR is to just move any optarg into the body.